examples/controlplane/controlplane.yaml

```
@@ -77,18 +77,20 @@ spec:
         "cloud": "AzurePublicCloud",
         "tenantID": "${AZURE_TENANT_ID}",
         "subscriptionID": "${AZURE_SUBSCRIPTION_ID}",
```

```suggestion
        "subscriptionId": "${AZURE_SUBSCRIPTION_ID}",
```

reference https://github.com/kubernetes/kubernetes/blob/695c3061dd92a6b6950f8adf0341ceb4a8dd44d7/staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth.go#L61